release/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash 
+
+set -e
+
+source ./version.sh
+
+LIGHT_GREEN='\033[1;32m'
+RED='\033[31m'
+NC='\033[0m'
+
+echo -e "${LIGHT_GREEN}Exporting environment variables in vars.sh${NC}"
+cd ../docker || exit 1
+if [ -f "vars.sh" ]; then
+    source vars.sh
+else
+  echo "${RED}ERROR: Missing vars.sh file${NC}"
+  exit 1
+fi
+
+echo -e "${LIGHT_GREEN}Stopping running containers${NC}"
+docker-compose down
+
+echo -e "${LIGHT_GREEN}Building and starting dependencies${NC}"
+docker-compose up -d activemq mongodb wiremock mock-mhs-adaptor
```

Comment:
GPC consumer adaptor doesn't use any of these dependencies. Are they even in the docker-compose file?

Reply:
Going to delete them